Remotion/Web/Development.WebTesting.ControlObjects/WebTabStripTabDefinition.cs

```diff
@@ -60,5 +62,10 @@ public bool IsDisabled
     {
       get { return _isDisabled; }
     }
+
+    public string AccessKey
+    {
+      get { return _accessKey; }
```

Comment:
This would either return `string.Empty` or `"AccessKey"` -> please check the origin of the value.